submissions/zalevskyi/html-css-popup/popup.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang='en' class='full_screen_hight'>
+    <head>
+        <title>Pop-up CSS only</title>
```

Comment:
Add meta tag for mobile devices.

Reply:
Hi, thank you for the review
Viewport meta tag added and slightly changed css with min-width for mobile devices.